src/cleaner.rs

```diff
@@ -207,6 +207,11 @@ pub fn write_buffer(doc: &Document, opt: &WriteOptions, buf: &mut Vec<u8>) {
     doc.write_buf_opt(opt, buf);
 }
 
+pub fn write_stdout(data: &[u8]) -> Result<(), io::Error> {
+    io::stdout().write(&data).unwrap();
+    Ok(())
```

Comment:
You should remove either `Ok(())` or `unwrap()`, because this code has no meaning.